src/metabase/api/user.clj

```diff
@@ -204,6 +204,7 @@
         perms-query {:where [:and
                              [:= :archived false]
                              coll-ids-filter]}]
+    #_{:clj-kondo/ignore [:discouraged-var]}
```

Comment:
this is a bug https://github.com/metabase/metabase/issues/29438

Adam will fix it and use toucan2 in the fix.